packages/plugin-ext/src/main/node/plugin-deployer-impl.ts

```diff
@@ -83,8 +87,8 @@ export class PluginDeployerImpl implements PluginDeployer {
         // init resolvers
         await this.initResolvers();
 
-        // check THEIA_DEFAULT_PLUGINS or THEIA_PLUGINS env var
-        const defaultPluginsValue = process.env.THEIA_DEFAULT_PLUGINS || undefined;
+        // check the `--plugins` CLI option and THEIA_PLUGINS env var
+        const defaultPluginsValue = this.cliContribution.localDir();
```

Comment:
hello @kittaakos 
we still need to check for process.env.THEIA_DEFAULT_PLUGINS (it's a breaking change if not)

would be better to add another source `this.cliContribution.localDir();` and merge it to the two other sources (THEIA_PLUGINS and THEIA_DEFAULT_PLUGINS)

Reply:
What do you mean by merging? It is a string. Can you please clarify?

Reply:
Did you mean something like this?
```ts
const defaultPluginsValue = this.cliContribution.localDir() || process.env.THEIA_DEFAULT_PLUGINS ;
```

Reply:
I have updated the PR so that it is backward compatible. When the `THEIA_DEFAULT_PLUGINS` used instead of the CLI option, it will log a warning. Please review.